README.md

```diff
@@ -207,7 +207,8 @@ backends = [
   'dlib', 
   'mtcnn', 
   'retinaface', 
-  'mediapipe'
+  'mediapipe',
+  'yolov8n',
```

Comment:
IMO, detector name should be yolo instead of yolov8n

Reply:
YOLO evolve in all direction, so I'm not sure if it's ideal to not specify the version

Reply:
yolov8 instead of yolov8n?

Reply:
Ok with `yolov8`